docs/assemblies/con_provisioning-bare-metal-data-plane-nodes.adoc

```diff
@@ -0,0 +1,17 @@
+[id="con_provisioning-bare-metal-data-plane-nodes_{context}"]
```

Comment:
This repeats lot of stuff already have in https://openstack-k8s-operators.github.io/dataplane-operator/#_provisioning_baremetal_nodes